src/flp/szk.rs

```diff
@@ -308,14 +303,13 @@ where
 {
     /// The Type representing the specific FLP system used to prove validity of an input.
     pub(crate) typ: T,
-    algorithm_id: u32,
```

Comment:
We currently don't bind XOF computations to this, like we do in Prio3. See https://github.com/jimouris/draft-mouris-cfrg-mastic/issues/99.